src/dotnet/UI.Blazor.App/Services/Search/SearchUI.cs

```diff
@@ -97,15 +100,25 @@ private void NavbarUIOnSelectedGroupChanged(object? sender, NavbarGroupChangedEv
 
     public void Select(FoundItem foundItem)
     {
```

Comment:
Why SelectPrevious/SelectNext are implemented with using NavigateTo, but this method has completely different implemention?